ompi/mca/coll/tuned/coll_tuned_reduce_decision.c

```diff
@@ -31,6 +31,7 @@ static int coll_tuned_reduce_segment_size = 0;
 static int coll_tuned_reduce_max_requests;
 static int coll_tuned_reduce_tree_fanout;
 static int coll_tuned_reduce_chain_fanout;
+static int coll_tuned_reduce_knomial_radix = 4;
```

Comment:
Why `coll_tuned_reduce_knomial_radix` is 4 while `coll_tuned_allgather_radix` is 16 ?

Reply:
The coll_tuned_reduce_knomial_radix is intended for intranode reduce and coll_tuned_allgather_radix  is for internode allgather.  The default values are chosen based on allreduce latency results on EFA.  It is likely tied to internode, intranode p2p message latencies.

Reply:
that's good you said it here, because it is not clear from the code or from the comments. Please update accordingly.